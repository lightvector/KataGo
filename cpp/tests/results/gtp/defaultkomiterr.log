: Running with following config:
allowResignation = true
cudaUseFP16 = false
cudaUseNHWC = false
forDeterministicTesting = true
hasButton = false
koRule = POSITIONAL
lagBuffer = 1.0
logAllGTPCommunication = true
logFile = tests/results/gtp/defaultkomiterr.log
logSearchInfo = true
logSearchInfoForChosenMove = false
logTimeStamp = false
logToStderr = false
maxPlayouts = 10000
maxTimePondering = 60.0
maxVisits = 100
multiStoneSuicideLegal = false
nnRandSeed = forTesting
nnRandomize = false
numSearchThreads = 1
openclUseFP16 = false
ponderingEnabled = false
resignConsecTurns = 3
resignThreshold = -0.90
rootSymmetryPruning = false
scoringRule = TERRITORY
searchFactorAfterOnePass = 0.50
searchFactorAfterTwoPass = 0.25
searchFactorWhenWinning = 0.40
searchFactorWhenWinningThreshold = 0.95
searchRandSeed = forTesting
taxRule = NONE
trtUseFP16 = false

: GTP Engine starting...
: KataGo v1.16.4
: Using koPOSITIONALscoreTERRITORYtaxNONEsui0 rules initially, unless GTP/GUI overrides this
: Using 1 CPU thread(s) for search
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Initializing neural net buffer to be size 19 * 19 exactly
: Cuda backend thread 0: Found GPU NVIDIA RTX A5000 memory 25425608704 compute capability major 8 minor 6
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 19 nnYLen 19
: Initializing board with boardXSize 19 boardYSize 19
: Loaded config configs/gtp_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: hasButton = false
: Config override: koRule = POSITIONAL
: Config override: logDir = 
: Config override: logFile = tests/results/gtp/defaultkomiterr.log
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: multiStoneSuicideLegal = false
: Config override: nnRandSeed = forTesting
: Config override: nnRandomize = false
: Config override: numSearchThreads = 1
: Config override: openclUseFP16 = false
: Config override: rootSymmetryPruning = false
: Config override: rules = 
: Config override: scoringRule = TERRITORY
: Config override: searchRandSeed = forTesting
: Config override: taxRule = NONE
: Config override: trtUseFP16 = false
: Model name: g170-b6c96-s175395328-d26788732
: GTP ready, beginning main protocol loop
: Controller: genmove_debug b
: MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . @ . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreTERRITORYtaxNONEsui0komi6.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 90
NN batches: 90
NN avg batch size: 1
PV: C16 D3 Q17 Q3 Q15
Tree:
: T   1.40c W   0.23c S   0.39c ( +0.2 L  +0.4) N     100  --  C16 D3 Q17 Q3 Q15
---Black(^)---
C16 : T   2.82c W   1.43c S   0.60c ( +0.5 L  +0.6) LCB   -0.15c P  7.48% WF  19.1 PSV      26 N      19  --  C16 D3 Q17 Q3 Q15
Q3  : T   1.95c W   0.51c S   0.42c ( +0.3 L  +0.4) LCB   -2.83c P  8.95% WF  13.0 PSV      13 N      13  --  Q3 Q17 C4 D17 E4
Q17 : T   1.77c W   0.66c S   0.48c ( +0.4 L  +0.5) LCB   -4.04c P  8.75% WF  11.0 PSV      11 N      11  --  Q17 D3 D17 Q3 Q15
R4  : T   1.24c W  -0.65c S   0.21c ( -0.0 L  +0.3) LCB   -5.67c P  9.09% WF  11.0 PSV      10 N      11  --  R4 D3 Q17 D17
C4  : T   1.35c W   0.45c S   0.38c ( +0.2 L  +0.3) LCB   -6.77c P  8.52% WF  10.0 PSV       9 N      10  --  C4 Q17 D17 Q3 E4
D3  : T   1.17c W   0.27c S   0.42c ( +0.3 L  +0.5) LCB  -10.62c P  8.33% WF   8.0 PSV       8 N       8  --  D3 Q17 Q3 D17
R16 : T   1.38c W   0.20c S   0.39c ( +0.2 L  +0.4) LCB  -10.65c P  7.10% WF   8.0 PSV       8 N       8  --  R16 D3 D17 Q3 P16
D17 : T   0.67c W  -0.39c S   0.30c ( +0.1 L  +0.3) LCB   -7.87c P  7.53% WF   8.0 PSV       6 N       8  --  D17 Q17 C4 Q3 E4
Q4  : T  -0.59c W  -1.02c S   0.23c ( +0.0 L  +0.4) LCB  -28.75c P  7.11% WF   5.0 PSV       4 N       5  --  Q4 D3 Q17 D17
D4  : T  -0.01c W  -0.75c S   0.14c ( -0.1 L  +0.3) LCB  -43.26c P  5.74% WF   4.0 PSV       4 N       4  --  D4 D17 Q17

: = Q3
: GPU -1 finishing, processed 90 rows 90 batches
: All cleaned up, quitting
