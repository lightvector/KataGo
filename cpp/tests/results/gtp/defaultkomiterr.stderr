KataGo v1.8.0
Using koPOSITIONALscoreTERRITORYtaxNONEsui0 rules initially, unless GTP/GUI overrides this
Loaded config configs/gtp_example.cfg and/or command-line and query overrides
Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
Model name: g170-b6c96-s175395328-d26788732
GTP ready, beginning main protocol loop
MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . @ . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .

koPOSITIONALscoreTERRITORYtaxNONEsui0komi6.5
Time taken: ###
Root visits: 100
New playouts: 100
NN rows: 95
NN batches: 95
NN avg batch size: 1
PV: Q3 Q17 C4
Tree:
: T   0.90c W  -0.36c S   1.25c ( -0.0 L  +0.2) N     100  --  Q3 Q17 C4
---Black(^)---
Q3  : T   1.23c W   0.01c S   1.21c ( +0.1 L  +0.3) LCB   -8.47c P  8.95% WF  7.21% PSV      15 N      10  --  Q3 Q17 C4
R4  : T   0.78c W  -0.56c S   1.34c ( -0.1 L  +0.2) LCB  -11.05c P  9.08% WF  7.15% PSV      10 N      10  --  R4 D3 Q17
Q17 : T   1.50c W  -0.02c S   1.53c ( +0.1 L  +0.3) LCB  -10.67c P  8.75% WF  7.24% PSV      10 N      10  --  Q17 D3 D17
C4  : T   1.15c W  -0.31c S   1.47c ( -0.1 L  +0.1) LCB  -12.46c P  8.52% WF  7.20% PSV      10 N      10  --  C4 Q17 D17
D3  : T   1.27c W  -0.05c S   1.32c ( +0.1 L  +0.3) LCB   -9.84c P  8.33% WF  7.21% PSV      10 N      10  --  D3 Q17 Q3
D17 : T   0.93c W  -0.49c S   1.42c ( -0.1 L  +0.1) LCB  -18.26c P  7.53% WF  7.17% PSV       8 N       8  --  D17 D3 Q17
C16 : T   1.74c W   0.26c S   1.48c ( +0.1 L  +0.3) LCB  -16.94c P  7.47% WF  7.26% PSV       8 N       8  --  C16 D3 Q17
R16 : T   2.23c W   0.53c S   1.69c ( +0.3 L  +0.4) LCB  -14.56c P  7.10% WF  7.31% PSV       8 N       8  --  R16 D3 D17
Q4  : T  -0.03c W  -0.95c S   0.92c ( -0.1 L  +0.2) LCB  -24.43c P  7.11% WF  7.08% PSV       6 N       6  --  Q4 D3 Q17
D4  : T   0.68c W  -0.49c S   1.17c ( -0.0 L  +0.3) LCB  -25.13c P  5.74% WF  7.15% PSV       6 N       6  --  D4 D17 Q17
